docs/current_docs/manuals/developer/go/snippets/services/persist-service-state/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"context"
+)
+
+type MyModule struct{}
+
+func (m *MyModule) RedisService(ctx context.Context) (string, error) {
+	// create Redis service container
```

Comment:
Changed

Reply:
Sorry, it got my wires crossed here! I'm used to documenting a function inside it (python), but in Go and TS it's the line above it. Even in Python it has to be a multi-line string (`"""`), rather than an actual comment (`#`).